docs/Configuration.md

```diff
@@ -179,7 +179,7 @@ e.g.:
     start_lines: "3"
     line_count: 11
     detected_license: "GPL-2.0-only"
-    reason: CODE,
+    reason: CODE
```

Comment:
As mentioned [here](https://github.com/heremaps/oss-review-toolkit/pull/1856#discussion_r330069585), I believe we should quote this (even if not technically required in this case).

Reply:
Done.